examples/scheduled_api_call/main.tf

```diff
@@ -0,0 +1,29 @@
+# ---------------------------------------------------------------------------------------------------------------------
```

Comment:
Before working on the feedback, please pull the branch as I've pushed already some slights refactorings by myself!